src/commands/adivinhar.js

```diff
@@ -81,37 +94,66 @@ async function sendGameMessageAndResults(interaction) {
 		'line5': `${square['gray'].repeat(5)}`,
 		'line6': `${square['gray'].repeat(5)}`,
 	};
-	// A tabela do jogo, gerada dinamicamente
+
 	function returnGameTable() {
-		return Object.values(gameMessage).map(line => line).join('\n');
+		if (!usersTries.find(user => user.id === interaction.user.id)) {
+			return Object.values(gameMessage).map(line => line).join('\n');
+		}
+		else {
+			const userTries = usersTries.find(user => user.id === interaction.user.id).attempts;
+			for (let i = 0; i < 6; i++) {
+				if (userTries[i] === undefined) {
```

Comment:
```suggestion
				if (!userTries[i]) {
``` 

Just a suggestion, there's no problem with the current code. Although there's an [eslint rule](https://eslint.org/docs/rules/no-undefined#no-undefined) against this (which is disabled).